zookeeper-api/src/main/java/org/apache/helix/zookeeper/routing/RoutingDataManager.java

```diff
@@ -166,8 +172,18 @@ public MetadataStoreRoutingData getMetadataStoreRoutingData(
 
   /**
    * Clears the statically-cached routing data and private fields.
+   * @param isForcedReset - if true, ignore throttle settings
    */
-  public synchronized void reset() {
+  public synchronized void reset(boolean isForcedReset) {
```

Comment:
all the methods in this change are calling with "true". and pure reset() is private. so do we really have a usecase with "false" ?

Reply:
reset(false) is used in FederatedZkClient and DedicatedZkClient.